apps/expo/app.config.ts

```diff
@@ -47,7 +46,7 @@ const defineConfig = (_: ConfigContext): ExpoConfig => ({
     },
   },
   android: {
-    package: "dev.mozzius.graysky",
+    package: process.env.APP_ID,
```

Comment:
Could you default it to `dev.mozzius.graysky` if a value for APP_ID is unspecified?

Reply:
ok, done. And for ios:bundleIdentifier too